ca-operator/ca-operator-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: ca-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ca-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
+subjects:
+- kind: ServiceAccount
+  name: ca-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ca-operator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: ca-operator
+  template:
+    metadata:
+      labels:
+        app: ca-operator
+    spec:
+      containers:
+      - image: ${CLI_IMAGE}
+        imagePullPolicy: IfNotPresent
+        name: ca-operator
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |-
+          #!/bin/bash
+          set -e
```

Comment:
we might revisit/enhance this to not error out with the configmap does not exist, which results pod restart and backoff